insights/parsers/du_usage.py

```diff
@@ -0,0 +1,55 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command.  The base class
+``DiskUsage`` provides all of the functionality for all classes.
+Data is avaliable as rows of the output contained
+in one ``Record`` object for each line of output.
+
+Sample input data for the ``du -sh`` command looks like::
+22G     /var/lib/mongodb
+
+"""
+
+from .. import Parser, parser, LegacyItemAccess
+
+
+def parse_du_lines(du_content):
```

Comment:
If no other modules will use this function, better add a `_` as a prefix to indicate it's a private/local interface.

Reply:
I see no real point in having a separate parse function when you can just include this code in the `parse_content` method of the class.

Reply:
done.